cmd/heimdalld/init.go

```diff
@@ -50,6 +50,12 @@ func initCmd(ctx *server.Context, cdc *codec.Codec) *cobra.Command {
 
 			WriteDefaultHeimdallConfig(filepath.Join(config.RootDir, "config/heimdall-config.toml"), helper.GetDefaultHeimdallConfig())
 
+			// copy toml files for each network chain into config folder
+			err = helper.CopyNetworkChainTomlsToConfigDir(filepath.Join(config.RootDir, "config"), helper.GetValidNetworkChains())
```

Comment:
what is the goal of this?

Reply:
this copies mainnet.toml and mumbai.toml from build/app directory to config directory. If we hardcode things, this is not needed